provision.sh

```diff
@@ -71,7 +71,8 @@ echo "---------------------------------------------"
 su - vagrant -c "/usr/local/bin/virtualenv /home/vagrant/.virtualenvs/${ENV_NAME} --python=/usr/bin/python3 && \
     /home/vagrant/.virtualenvs/${ENV_NAME}/bin/pip install -r /vagrant/requirements.txt"
 
-su - vagrant -c "cp /vagrant/.bashrc /home/vagrant/.bashrc"
+su - vagrant -c "cp /vagrant/.bashrc /home/vagrant/"
+su - vagrant -c "cp /vagrant/gitconfig.sh /home/vagrant/gitconfig"
```

Comment:
I edited gitconfig.sh on Mac OS with Atom. Do I still need dos2unix? Is it best practice to include this at all times? I read online that this is only relevant for people who developed the code on Windows.